tests/core/test_statsd.py

```diff
@@ -18,7 +19,20 @@ def _private_method(self):
         pass
 
 
-@unittest.skipIf(not statsd.statsd_module, "statsd is not installed.")
+class StatsDMissing(unittest.TestCase):
+    def setUp(self):
+        self.previous = statsd.statsd_module
+        statsd.statsd_module = None
+
+    def tearDown(self):
+        statsd.statsd_module = self.previous
+
+    def test_client_instantiation_raises_properly(self):
+        with self.assertRaises(ConfigurationError):
+            statsd.Client('localhost', 1234, 'prefix')
+
```

Comment:
Maybe a test that makes sure that the client instantiation doesn't raise if the statsd is not configured and not installed, and neither if statsd is configured and installed?